### PR TITLE
docs: `integrations` reference updates 4

### PR DESCRIPTION
**Pull request by leo-gan**
Added missed references; missed provider pages.

### CODE REVIEW

**Comment by leo-gan** [comment]
@eyurtsev Could you, please, check the Vercel error message? I don't have an access for it.

**Comment by leo-gan** [comment]
@ccurme Can you review, please?

File: docs/docs/integrations/providers/asknews.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# AskNews
+
+>[AskNews](https://asknews.app/) infuses any LLM with the latest global news
```
Comment (ccurme) [inline docs/docs/integrations/providers/asknews.mdx:3]: I don't think we should be copy/pasting material from other websites into the LangChain docs, especially as the attribution here is a little unclear-- we link to https://asknews.app but this appears lifted from https://docs.asknews.app/en. I don't think formatting with a comment block is sufficient. IMO this should be a high-level summary and a link to their page for more information.

Comment (leo-gan) [reply docs/docs/integrations/providers/asknews.mdx]: Yes, I agree. Ideally, the company itself should create a page in the `providers/`. Maybe, we can send a request to all providers, asking them to provide us the "short company description"?
Now, it is mostly copy-past from the provider materials inside a comment block. It is better than no provider info at all. 

Sometimes, it takes a lot of time to find the company information. 
Try to find information about the provider of the EmbaasEmbeddings class, for example.

**Review by ccurme — APPROVED** [review]

**Comment by ccurme** [comment]
Thank you for your help with this!

**Pull request merged**